test/test_pip.py

```diff
@@ -166,3 +166,13 @@ def test_upgrade(self):
     rm -rf /var/lib/apt/lists/*
 RUN pip --no-cache-dir install --upgrade "pip < 21.0" && \
     pip --no-cache-dir install hpccm''')
+
+    @ubuntu
+    @docker
+    def test_install_args(self):
+        """install_args option"""
+        p = pip(ospackages=[], packages=['hpccm'],
+                install_args=['--index-url', 'https://my-index.com'])
```

Comment:
Should this technically be a single argument, `--index-url https://my-index.com`?